spark_pipeline_framework/transformers/fhir_sender/v1/fhir_sender_processor.py

```diff
@@ -66,9 +67,60 @@ def send_partition_to_server(
         request_id_list: List[str] = []
         responses: List[Dict[str, Any]] = []
         if FhirSenderOperation.operation_equals(
+            operation, FhirSenderOperation.FHIR_OPERATION_PATCH
+        ):
+            for item in json_data_list:
+                item_value = json.loads(item["value"])
+                payload = [
+                    json.loads(convert_dict_to_fhir_json(payload_item))
+                    for payload_item in item_value["payload"]
+                ]
+                patch_result: Optional[Dict[str, Any]] = update_json_bundle_to_fhir(
+                    obj_id=item_value["id"],
+                    json_data=json.dumps(payload),
+                    server_url=server_url,
+                    operation=operation,
+                    validation_server_url=validation_server_url,
+                    resource=resource_name,
+                    logger=logger,
+                    auth_server_url=auth_server_url,
+                    auth_client_id=auth_client_id,
+                    auth_client_secret=auth_client_secret,
+                    auth_login_token=auth_login_token,
+                    auth_scopes=auth_scopes,
+                    auth_access_token=auth_access_token,
+                    additional_request_headers=additional_request_headers,
+                    log_level=log_level,
+                )
+                if patch_result:
+                    responses.append(patch_result)
+        elif FhirSenderOperation.operation_equals(
+            operation, FhirSenderOperation.FHIR_OPERATION_PUT
+        ):
+            for item in json_data_list:
+                item_value = json.loads(item["value"])
+                put_result: Optional[Dict[str, Any]] = update_json_bundle_to_fhir(
+                    obj_id=item_value["id"],
+                    json_data=convert_dict_to_fhir_json(item_value),
+                    server_url=server_url,
+                    operation=operation,
+                    validation_server_url=validation_server_url,
+                    resource=resource_name,
+                    logger=logger,
+                    auth_server_url=auth_server_url,
+                    auth_client_id=auth_client_id,
+                    auth_client_secret=auth_client_secret,
+                    auth_login_token=auth_login_token,
+                    auth_scopes=auth_scopes,
+                    auth_access_token=auth_access_token,
+                    additional_request_headers=additional_request_headers,
+                    log_level=log_level,
+                )
+                if put_result:
+                    responses.append(put_result)
```

Comment:
This may also contain error response as returned from update_json_bundle_to_fhir.

Reply:
yes...that will get processed down in line no. 247